replay-protected-sighash.md

```diff
@@ -1,12 +1,13 @@
-# BUIP-HF Digest for replay protected signature verification accross hard forks
+# BUIP-HF Digest for replay protected signature verification across hard forks
 
 Version 1.0, 2017-06-14
+Version 1.1, 2017-06-14
 
 ## Abstract
 
 This document describes proposed requirements and design for a reusable signing mechanism ensuring replay protection in the event of a hard fork. It provides a way for users to create transactions which are invalid on forks lacking support for the mechanism and a fork-specific ID.
 
-The proposed digest algorithm is adapted from [BIP143](https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki) as it minimizes redundant data hashing in verification, covers the input value by the signature [1] and is already implemented in a wide variety of applications.
+The proposed digest algorithm is adapted from [BIP143][1] as it minimizes redundant data hashing in verification, covers the input value by the [signature][2] and is already implemented in a wide variety of applications.
```

Comment:
I would move the reference [2] to the end of this sentence, because it is intended to span the entire

> minimizes redundant data hashing in verification, covers the input value